README.md

```diff
@@ -450,6 +477,49 @@ You can also use `exported_resources` directly in a test:
 it { expect(exported_resources).to contain_file('foo') }
 ```
 
+#### Testing applications
+
+Applications in some ways behave as defined resources, but are more complex so
+require a number of elements already documented above to be combined for testing.
+
+A full example of the simplest rspec test for a single component application:
+
+```ruby
+require 'spec_helper'
+
+describe 'orch_app' do
+  before(:each) do
+    Puppet[:app_management] = true
```

Comment:
Wouldn't you want this enabled at all times? Is there a reason to not have it enabled?